controllers/apply/standard-proposal/form.js

```diff
@@ -216,8 +216,8 @@ module.exports = function({ locale = 'en', data = {} } = {}) {
             cy: ''
         }),
         startLabel: localise({
-            en: 'Start',
-            cy: 'Dechrau'
+            en: 'Start your application',
```

Comment:
I'm not sure if we're using the word "application" here, but it can't just be this:
![image](https://user-images.githubusercontent.com/394376/68605826-03515c00-04a5-11ea-9b85-df74299b7de8.png)